client/app/src/main/java/com/microsoft/research/karya/ui/accesscode/AccessCodeFragment.kt

```diff
@@ -102,24 +87,35 @@ class AccessCodeFragment : Fragment(R.layout.fragment_access_code) {
     checkAccessCode(accessCode)
   }
 
-  private fun showSuccessMessage() {
+  private fun onAccessCodeVerified(successResult: Result.Success<Int>, accessCode: String) {
+    hideLoading()
+    showSuccessUi()
+    lifecycleScope.launch { updateLanguagePreference(successResult.value) }
+    WorkerInformation.creation_code = accessCode
```

Comment:
We are still using WorkerInformation?

Reply:
This is the other branch, it does not have any of the db changes